src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -777,6 +779,14 @@ export class FormTransferTransactionTs extends FormTransactionBase {
 
         this.isMounted = true;
     }
+
+    /**
+     * checks if account balance is enough to make a transaction
```

Comment:
```suggestion
     * Checks if account balance is enough to make a transaction
```

This method only seems to store a value, it does not look like it is "checking" anything, as the comment suggests. The name of the method makes me think this is an event handler. Shouldn't the comment say something like "Called when balance needs to be checked" ?